modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -22,11 +24,26 @@ class Test{{classname}}(unittest.TestCase):
     def tearDown(self):
         pass
 
-    def test{{classname}}(self):
+    def make_instance(self, include_optional):
```

Comment:
Can you add a description of the include_optional parameter to this docstring?
It will help our users to know that it is a boolean and if we set it to False only required params will be included, and if set to True then required + optional params will be included

Reply:
Done